src/App.js

```diff
@@ -1,9 +1,12 @@
 import React, { Component } from "react";
-
+import "./css/App.scss";
 class App extends Component {
 	render() {
 		return (
-			<div><h1>Authors Haven </h1></div>
+			<div className="app">
+				<h1>Authors Haven </h1>
+				<h2 className='mission'>A Social platform for the creative at heart.</h2>
+			</div>
 		);
 	}
 }
```

Comment:
Let's create a folder called `assets` in the src directory where we will put all the external dependencies. In the assets folder, you can then place the `css`folder you created.

Reply:
Noted

Reply:
You also need to maintain your indentation to 2 spaces, as you have done in the sass files

Reply:
@LehruAngela in the eslint config file. We setup indentation using tabs, so whenever I put 2 spaces indentation, it complains. Maybe I should change the Indentation rule!

Reply:
Let me make that change